backupstoragelocation.md

```diff
@@ -46,6 +46,19 @@ spec:
     # Optional.
     subscriptionId: my-subscription
 
+    # URI of the blob endpoint of the storage account. 
+    # 
+    # Optional. This will ensure that velero uses the provided URI to communicate to the Storage Account, 
+    # and it will not try to fetch the Endpoint by making an ARM call.
+    # If this field is provided then resourceGroup, subscriptionId can be left empty
```

Comment:
will also update the readme.

Reply:
Thanks @anshulahuja98 , Let's also mention the limitation for `useAAD` in this doc and the readme: https://github.com/vmware-tanzu/velero/issues/5984

Reply:
Updated the README would request you to review once again.